### PR TITLE
Updated Northeastern University CS faculty with new additions and retirements

### PR DESCRIPTION
**Pull request by grobalex**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

0. Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv"). **IF YOU DO NOT DO THIS, YOUR COMMIT WILL BE SUMMARILY REJECTED.**

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings-[a-z].csv`._ **Do not modify `csrankings.csv`, which is auto-generated.** **DO NOT MODIFY `csrankings-[0-9].csv`, WHICH ARE BEING DEPRECATED. USE `csrankings-[a-z].csv` INSTEAD.**

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [X] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [X] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [X]  Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**; please **do not** use the numeric suffixes, which are being obsoleted); include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [X] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [X]  If the institution you are adding is not in the US,
update `country-info.csv`.

**(Advanced) Quick contribution via a shallow clone** 

A full clone of the CSrankings repository is almost 2GB.  To contribute a change without creating a full local clone of the CSrankings repo, you can do a _shallow clone_.  To do so, follow these steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)
2. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.
3. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to do another contribution and some time has passed, perform steps 1-3 again, creating a fresh fork and shallow clone.

### CODE REVIEW

File: csrankings-a.csv [hunk 1]
```diff
@@ -291,7 +291,6 @@ Aggelos Pikrakis,University of Piraeus,https://www.unipi.gr/unipi/en/pikrakis.ht
 Agma J. M. Traina,USP-ICMC,http://icmc.usp.br/pessoas?id=4170385,v2-C4i4AAAAJ
 Agma Juci Machado Traina,USP-ICMC,http://icmc.usp.br/pessoas?id=4170385,v2-C4i4AAAAJ
 Agnar Aamodt,NTNU,https://www.ntnu.edu/employees/agnar,-3L6uJMAAAAJ
-Agnes Hui Chan,Northeastern University,http://www.ccs.neu.edu/home/ahchan,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings-a.csv:296]: Emerita? If so, please add to `old/emeritus.csv`.

File: csrankings-g.csv [hunk 2]
```diff
@@ -643,7 +643,7 @@ Gregoris Mentzas,NTUA,http://imu.ntua.gr/users/gmentzas,WdybrQYAAAAJ
 Gregory B. White,University of Texas at San Antonio,http://www.cs.utsa.edu/facstaff,yT6jxa8AAAAJ
 Gregory Bodwin,University of Michigan,https://bodwin.engin.umich.edu,NWpUSd4AAAAJ
 Gregory Butler,Concordia University,http://www.cs.concordia.ca/~gregb,K2vSwvQAAAAJ
-Gregory D. Abowd,Georgia Institute of Technology,http://ubicomp.cc.gatech.edu/gregory-d-abowd,gb8sbdcAAAAJ
+Gregory D. Abowd,Northeastern University,https://coe.northeastern.edu/people/abowd-gregory/,gb8sbdcAAAAJ
```
Comment (emeryberger) [inline csrankings-g.csv]: Please explain how they meet the inclusion criteria (not in CS).

File: csrankings-h.csv [hunk 3]
```diff
@@ -295,7 +295,6 @@ Harold T. Wareham,Memorial University of Newfoundland,https://www.mun.ca/compute
 Harold Vincent Poor,Princeton University,http://ee.princeton.edu/people/faculty/h-vincent-poor,NOSCHOLARPAGE
 Harri Lähdesmäki,Aalto University,https://users.ics.aalto.fi/harrila,NOSCHOLARPAGE
 Harrie Oosterhuis,Radboud University,https://harrieo.github.io,e9JynrAAAAAJ
-Harriet J. Fell,Northeastern University,http://www.ccs.neu.edu/home/fell,glFPEWIAAAAJ
```
Comment (emeryberger) [inline csrankings-h.csv:300]: Emerita?

File: csrankings-j.csv [hunk 4]
```diff
@@ -403,7 +403,7 @@ Jan van Leeuwen,Utrecht University,https://www.uu.nl/staff/JvanLeeuwen1,7s_pSa4A
 Jan-J. Rückmann,University of Bergen,https://www.uib.no/en/persons/Jan-Joachim.R%C3%BCckmann,NOSCHOLARPAGE
 Jan-Michael Frahm,University of North Carolina,http://frahm.web.unc.edu,3YOe4NMAAAAJ
 Jan-Philipp Steghöfer,Chalmers/GU,https://www.chalmers.se/en/staff/Pages/jan-philipp-steghofer.aspx,NOSCHOLARPAGE
-Jan-Willem van de Meent,Northeastern University,http://www.robots.ox.ac.uk/~jwvdm,CX9Lu38AAAAJ
+Jan-Willem van de Meent,Northeastern University,https://www.ccs.neu.edu/home/jwvdm/,CX9Lu38AAAAJ
```
Comment (emeryberger) [inline csrankings-j.csv]: In fact, should change affiliation: "As of 1 September, I will start at the University of Amsterdam and will be on leave from Northeastern University.". Please update accordingly.

File: csrankings-k.csv [hunk 5]
```diff
@@ -391,7 +391,6 @@ Kemal Akkaya,Florida International University,http://web.eng.fiu.edu/kakkaya,63H
 Kemal Oflazer,Carnegie Mellon University,https://www.andrew.cmu.edu/user/ko,JLk0hs8AAAAJ
 Kemao Qian,Nanyang Technological University,http://research.ntu.edu.sg/expertise/academicprofile/Pages/StaffProfile.aspx?ST_EMAILID=MKMQIAN,JCehBDoAAAAJ
 Ken Anderson 0001,University of Colorado Boulder,http://www.cs.colorado.edu/~kena,wOO8rEMAAAAJ
-Ken Baclawski,Northeastern University,http://www.ccs.neu.edu/home/kenb,Vwi6Ya8AAAAJ
```
Comment (emeryberger) [inline csrankings-k.csv:395]: Emeritus, please update emeritus file.

File: csrankings-l.csv [hunk 6]
```diff
@@ -50,7 +50,6 @@ Lap-Fai Yu,George Mason University,http://www.cs.umb.edu/~craigyu,p_Un9ssAAAAJ
 Larisa N. Soldatova,Goldsmiths University of London,https://www.gold.ac.uk/computing/people/soldatova-larisa,wUFUS-oAAAAJ
 Larissa A. Meinicke,University of Queensland,http://researchers.uq.edu.au/researcher/644,oMK7xR4AAAAJ
 Larissa Meinicke,University of Queensland,http://researchers.uq.edu.au/researcher/644,oMK7xR4AAAAJ
-Larry A. Finkelstein,Northeastern University,http://www.ccis.northeastern.edu/people/larry-finkelstein,I4iKZSkAAAAJ
```
Comment (emeryberger) [inline csrankings-l.csv:54]: Emeritus.

File: csrankings-m.csv [hunk 7]
```diff
@@ -2028,7 +2028,6 @@ Mitchell D. Theys,University of Illinois at Chicago,https://www.cs.uic.edu/Theys
 Mitchell L. Neilsen,Kansas State University,http://www.cis.ksu.edu/~neilsen,NOSCHOLARPAGE
 Mitchell Marcus,University of Pennsylvania,https://www.cis.upenn.edu/~mitch,niE2OkIAAAAJ
 Mitchell P. Marcus,University of Pennsylvania,https://www.cis.upenn.edu/~mitch,niE2OkIAAAAJ
-Mitchell Wand,Northeastern University,http://www.ccs.neu.edu/home/wand,Y0E0lsgAAAAJ
```
Comment (emeryberger) [inline csrankings-m.csv:2038]: Emeritus?

File: csrankings-r.csv [hunk 8]
```diff
@@ -656,7 +656,6 @@ Richard P. Brent,Australian National University,http://wwwmaths.anu.edu.au/~bren
 Richard P. Martin,Rutgers University,https://www.cs.rutgers.edu/~rmartin,kPyVN7AAAAAJ
 Richard P. Wildes,York University,http://www.cse.yorku.ca/~wildes,1vEw_kgAAAAJ
 Richard Peng,Georgia Institute of Technology,http://www.cc.gatech.edu/~rpeng,NOSCHOLARPAGE
-Richard Rasala,Northeastern University,http://www.ccs.neu.edu/home/rasala,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings-r.csv:665]: Emeritus?

File: csrankings-v.csv [hunk 9]
```diff
@@ -194,7 +194,6 @@ Victor de Boer,VU Amsterdam,http://www.victordeboer.com,9Ck790YAAAAJ
 Victoria Interrante,University of Minnesota,http://www-users.cs.umn.edu/~interran,vQQOZVgAAAAJ
 Victoria Stodden,Univ. of Illinois at Urbana-Champaign,https://ischool.illinois.edu/people/victoria-stodden,LWw60SgAAAAJ
 Vida Dujmovic,University of Ottawa,http://cglab.ca/~vida,NOSCHOLARPAGE
-Viera K. Proulx,Northeastern University,http://www.ccs.neu.edu/home/vkp,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings-v.csv:197]: Emerita?

File: csrankings-w.csv [hunk 10]
```diff
@@ -324,7 +324,6 @@ William Allen,Florida Institute of Technology,http://www.fit.edu,NOSCHOLARPAGE
 William Armitage,University of South Florida,http://www.usf.edu/engineering/cse/people,NOSCHOLARPAGE
 William Brent Seales,University of Kentucky,https://www.cs.uky.edu/people/faculty/bseales,jk09W8sAAAAJ
 William C. Regli,University of Maryland - College Park,https://www.cs.umd.edu/people/regli,-MSUomIAAAAJ
-William D. Clinger,Northeastern University,http://www.ccis.northeastern.edu/people/william-d-clinger,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings-w.csv:329]: Emeritus, please update.

**Review by emeryberger — CHANGES_REQUESTED** [review]
Mostly questions about departures - please update `old/emeritus.csv` with these removed entries. Thanks.

File: csrankings-j.csv (hunk 4 above)
Comment (grobalex) [reply csrankings-j.csv]: Updated Jan-Willem's affiliation to the University of Amsterdam

File: csrankings-g.csv (hunk 2 above)
Comment (grobalex) [reply csrankings-g.csv]: Gregory has a Courtesy Appointment in the Khoury College of Computer Sciences, with PhD student advising privileges. Please see https://www.khoury.northeastern.edu/people/gregory-abowd/

**Pull request closed without merging**